Comme vous pouvez le constater,
J'ai réussi l'exercice !
Mais avant de me donner tous les points, il faut vérifier qu'il y a bien la version de pip, le fichier log, la local_lib, le dossier et le fichier dedans avec ce message...